client/src/utils/navigation/navigation.yml

```
@@ -513,6 +513,13 @@ histories:
       share_with_input: '.share_with_view input'
       make_accessible: '.make-accessible label'
       make_publishable: '.make-publishable label'
+      tab_make_private: '[aria-controls="history-make-private-tab"]'
+      tab_sharing: '[area-controls="#history-sharing-tab"]'
```

This selector is incorrect and not in use:
```suggestion
      tab_sharing: '[area-controls="history-sharing-tab"]'
```

---

I'll drop the selector - thanks for the catch!